lib/mayaUsd/fileio/translators/skelBindingsProcessor.h

```diff
@@ -62,6 +64,8 @@ class UsdMaya_SkelBindingsProcessor
     using _Entry = std::pair<SdfPath, TfToken>;
 
     std::unordered_map<SdfPath, _Entry, SdfPath::Hash> _bindingToSkelMap;
+
+    SdfPath mRootPrimPath;
```

Comment:
The coding standard in MayaUSD is to use a '_' prefix for member variables, like _bindingToSkelMap above.